content/docs/reference/spec/migration/buildpack-api-0.5-0.6.md

```diff
@@ -0,0 +1,39 @@
++++
+title="Buildpack API 0.5 -> 0.6"
++++
+
+<!--more-->
+
+This guide is most relevant to buildpack authors.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.6) for buildpack API 0.6 for the full list of changes and further details.
```

Comment:
Need to release it only after the spec is released (the link is currently broken).